### DISCUSSION TITLE
"add-certificate" page returns code 404

### DISCUSSION THREAD

**Discussion by juniwang** [polymind-inc/acmebot#415] [Q&A] [answered]
**Describe the bug**
Our Functions is deployed on Azure China. It has been working for a while. Not sure when it began but today when I try to open the webpage "https://{myFunctionName}.chinacloudsites.cn/**add-certificate**", it returns code 404. I tried to restart the Function, restart the trigger but no luck.  Could anyone help on this?

![image](https://user-images.githubusercontent.com/8814383/144528983-6ef3ef42-0ec4-42b1-9c33-326d18d85fc3.png)

And below is the config of the Function App:
![image](https://user-images.githubusercontent.com/8814383/144529262-d26fe758-7bd9-4383-af92-8be0089bd541.png)

**To Reproduce**

**Environment (please complete the following information):**
 - Certificate Type: [e.g. Zone Apex, Sub-domain, Wildcard]
 - Certificate Deploy Target: App Service

**Additional context**

**Comment by shibayan** [comment, marked as answer]
The `add-certificate` page has been deprecated. Only the `dashboard` is available now.
https://github.com/shibayan/keyvault-acmebot/wiki/Getting-Started#6-access-to-dashboard

**Reply by juniwang** [reply]
Thanks, it works
